types/mongoose/index.d.ts

```diff
@@ -727,7 +727,7 @@ declare module "mongoose" {
     minimize?: boolean;
     read?: string;
     /** defaults to true. */
-    safe?: boolean;
+    safe?: boolean | { w?: number | string; wtimeout?: number; j?: boolean };
```

Comment:
I think there are more options that can be added here as well, however I wasn't able to find it after a (very quick) search on Google and the MongoDB docs.

Perhaps we should extract that into an interface instead and see if we can find a full list of options?

The link provided here is dead, it seems: http://mongoosejs.com/docs/2.7.x/docs/schema-options.html

Reply:
I believe this is the documentation/options available for the `safe` option: https://docs.mongodb.com/manual/reference/write-concern/index.html